docs/fuel-101/intro-to-utxo.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Intro to UTXO
+category: Fuel 101
+---
+
+# Understanding UTXO
+
+The Unspent Transaction Output (UTXO) model, is a fundamental concept in Fuel's underlying blockchain architecture. Fuel tracks digital currency through transaction outputs, each representing a discrete piece of state that can be spent. This model is extended by stateful UTXOs to include additional data or conditions, enabling complex transactions and smart contract functionalities within a UTXO framework.
+
```

Comment:
It feels like there should be a section header here like "How do transactions work with a UTXO model?" or something